README.md

```diff
@@ -150,7 +151,7 @@ git clone https://github.com/django/django.git
 Enter the command:
 
 ```sh
-sudo pip install django
+pip install django
```

Comment:
Thanks! Shouldn't be the installation be without OS' rather than having Windows, Mac and Linux if we use the same command?